osu.Game.Rulesets.Osu/Skinning/Legacy/LegacySliderBody.cs

```diff
@@ -23,29 +23,32 @@ protected override Color4 GetBodyAccentColour(ISkinSource skin, Color4 hitObject
 
         private partial class LegacyDrawableSliderPath : DrawableSliderPath
         {
-            private const float shadow_portion = 1 - (OsuLegacySkinTransformer.LEGACY_CIRCLE_RADIUS / OsuHitObject.OBJECT_RADIUS);
-
-            protected new float CalculatedBorderPortion
-                // Roughly matches osu!stable's slider border portions.
-                => base.CalculatedBorderPortion * 0.77f;
-
             protected override Color4 ColourAt(float position)
             {
-                float realBorderPortion = shadow_portion + CalculatedBorderPortion;
-                float realGradientPortion = 1 - realBorderPortion;
+                // https://github.com/peppy/osu-stable-reference/blob/3ea48705eb67172c430371dcfc8a16a002ed0d3d/osu!/Graphics/Renderers/MmSliderRendererGL.cs#L99
+                const float aa_width = 3f / 256f;
```

Comment:
At the point we're putting the effort to replicate things this closely it begs the question why this isn't reproduced exactly from stable's

```csharp
float aa_width = Math.Min(Math.Max(0.5f / m_default_radius, 3.0f / 256.0f), 1.0f / 16.0f);
```

It's just not a constant 3/256 in stable. It could be more than that depending on circle size and screen resolution. This may result in inconsistent sharpness of the slider border.

For reference `m_default_radius` seems to be `hitObjectManager.HitObjectRadius * GameBase.GameField.Ratio` where

```csharp
        internal float Ratio { get { return Height / GameField.DEFAULT_HEIGHT; } }
        // where
        internal const int DEFAULT_HEIGHT = 384;
```

But I'm not super sure any of this is worth the effort.

Reply:
Good catch, I missed that part.

Reply:
See https://github.com/ppy/osu/pull/27383#issuecomment-1967552676